cmake/DetectOptions.cmake

```diff
@@ -502,9 +502,9 @@ endif()
 
 # AWS S3
 if(ADIOS2_USE_AWSSDK STREQUAL AUTO)
-    find_package(AWSSDK QUIET COMPONENTS s3)
+    find_package(AWSSDK 1.10.15 COMPONENTS s3)
```

Comment:
Unless there is an specif reason to use from 1.10.15, Maybe 1.10.0 is best here since it is less specific and includes every patch release of the >= 1.10.X.

Reply:
I suspect that Norbert took the easy way out and specified "something newer than what I developed with", as opposed to spending time digging into what the real minimum might be.   Maybe 1.10.0 works, if they did the right thing with the path releases, but did they?  Given that this isn't a stable, documented part of ADIOS that we want people to use, I wouldn't worry about this too much.  The purpose here is not to get more issues about AWSSDK until if and when we're ready for it.

Reply:
Fair enough. Approving too :)